external/storm-jdbc/src/test/java/org/apache/storm/jdbc/common/UtilTest.java

```diff
@@ -46,20 +46,7 @@ public void testBasic() {
 
     @Test
     public void testError() {
-        //This test is rather ugly, but it is the only way to see if the error messages are working correctly.
-        try {
-            Util.getJavaType(Types.REF);
-            fail("didn't throw like expected");
-        } catch (Exception e) {
-            assertEquals("We do not support tables with SqlType: REF", e.getMessage());
-        }
-
-        try {
-            Util.getJavaType(-1000);
-            fail("didn't throw like expected");
-        } catch (Exception e) {
-            assertEquals("Unknown sqlType -1000", e.getMessage());
-        }
-
+        assertThrows(RuntimeException.class, () -> Util.getJavaType(Types.REF));
```

Comment:
instead of RuntimeException, these should be Exception

Reply:
done